tensorboard/plugins/scalar/summary_v2.py

```diff
@@ -30,6 +30,34 @@
 def scalar(name, data, step=None, description=None):
     """Write a scalar summary.
 
+    See also `tf.summary.image`, `tf.summary.histogram`, `tf.summary.SummaryWriter`.
+
+    Logs simple numeric values for later analysis in TensorBoard.  Each summary
+    point is associated with an integral `step` value. This enables the incremental
+    logging of time series data.  A common usage of this API is to log loss during
+    training to produce a loss curve.
+
+    For example:
+
+    ```python
+    test_summary_writer = tf.summary.create_file_writer('test/logdir')
+    with test_summary_writer.as_default():
+        tf.summary.scalar('loss', 0.345, step=1)
+        tf.summary.scalar('loss', 0.234, step=2)
+        tf.summary.scalar('loss', 0.123, step=3)
+    ```
+
+    Multiple independent time series may be logged by giving each series a unique
+    `name` value.
+
+    See https://www.tensorflow.org/tensorboard/get_started for more examples
```

Comment:
I'm not 100% sure if bare links linkify automatically for tensorflow.org.  Maybe ask the docs people?  E.g. here's one example that doesn't: https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/python/distribute/tpu_strategy.py;l=246;drc=adf69bf65356f810fd4c98cd6d3914ca1d451ca0

Compare with rendering: https://www.tensorflow.org/api_docs/python/tf/distribute/TPUStrategy?version=nightly

Reply:
Updated with Markdown style link.